### PR TITLE
Select blackbox debug fields

### PR DESCRIPTION
**Pull request by haslinghuis**
Selecting blackbox debug fields was introduced in API 1.45 but not implemented in UI.

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
